feature/stn/instrument/index.js

```diff
@@ -27,6 +30,8 @@ var PUSH_STATE = 'pushState'
 
 // Turn on feature harvesting
 var loader = require('loader')
+loader = Object.assign({}, loader, {features: Object.assign(loader.features || {}, {stn: true})})
```

Comment:
We initialize the `features` property here
https://github.com/newrelic/newrelic-browser-agent/blob/main/loader/index.js#L53

But I see that the early return in loader/index.js changes what is exported. Maybe we should still export something - maybe just a flag that says the agent is disabled. That might be better than running the same protocol check in multiple files.

Reply:
Good call, this is the info i was looking for.

Reply:
I also just removed that and moved where the early return happens so that the object is no longer empty.